auth-web/src/views/auth/home/DecideBusinessView.vue

```diff
@@ -9,9 +9,8 @@
           <v-list-item-content>
             <v-list-item-subtitle class="list-item-text">
               {{item.text}}
-              <a href="https://www2.gov.bc.ca/gov/content?id=3E4E169B42DF43EEA19E96383F8FD628"
-               class="learn-more-link" target="_blank" rel="noopener noreferrer">
-                {{item.linkText}}
+              <a class="list-item-link" :href="item.url" target="_blank" rel="noopener noreferrer">{{item.linkText}}
+                <v-icon class="link-icon mb-1" small color="#1a5a96">{{item.icon}}</v-icon>
```

Comment:
Can a named colour be used here (as we've done in other projects)?

(not a deal-breaker)

Reply:
It can be, but we would have to define that on Veutify in `main.ts`. 
That being said, this colour doesn't jive any named colours we have in our other UI's and i don't like the idea of having the same name (ie `primary` or something in this case) be multiple colours across the UI's. 
Thoughts?

Reply:
Leave it. That sounds less ugly than changing it.